pegasus/sites.v3/code.org/public/privacy.md

```diff
@@ -156,27 +159,27 @@ Code.org does not directly collect or store payment instruments. Donations to Co
 
 ### Exceptions
-If a student contacts us via our customer support pages, we will store and use their email address in order to respond to them. If a student over the age of 13 signs our online petitions, or a student creates a teacher account on Code.org or offers to help us as a software industry volunteer, we will store and use their email address just as we do for other advocates of our nonprofit mission.
+If a User contacts us via our customer support pages, we will store and use their email address in order to respond to them. If a User over the age of 13 signs our online petitions, or a User creates a Teacher account on Code.org or offers to help us as a software industry volunteer, we will store and use their email address just as we do for other advocates of our nonprofit mission.
 
-For a small minority of our students, and only if they are over the age of 18, we may offer the opportunity to participate in a "longitudinal" study to understand the multi-year impact of learning computer science. Participation in such a study would be entirely optional. Students who receive an offer and choose to participate will be asked to provide their contact information (email address and optionally other forms of contact that may be more convenient for the student). This contact information will not be shared with third parties, nor used in any way outside the purpose of such a study – to ask students to participate in surveys. If we learn that we have inadvertently collected this information from a child under 18, we will delete such information immediately.
+For a small minority of our Students, and only if they are over the age of 18, we may offer the opportunity to participate in a "longitudinal" study to understand the multi-year impact of learning computer science. Participation in such a study would be entirely optional. Students who receive an offer and choose to participate will be asked to provide their contact information (email address and optionally other forms of contact that may be more convenient for the Student). This contact information will not be shared with third parties, nor used in any way outside the purpose of such a study – to ask Students to participate in surveys. If we learn that we have inadvertently collected this information from a Student under 18, we will delete such information immediately.
 
 ## Children Under the Age 13
 
 Code.org courses are designed to be used by Users of all ages, including children under the age of 13 with the involvement of and pre-authorization by their school, teacher, parent or legal guardian. In order to use features that involve uploading custom media files or sending messages within the school, we obtain prior consent from the student’s teacher, other authorized school representative, or, in some cases, the parent or legal guardian.
```

Comment:
- pre-authorization by their school, `Teacher`, parent
- consent from the `Student’s Teacher`